tests/integration/test_permissions.py

```diff
@@ -84,6 +84,7 @@ def sample_record(app):
     db.session.delete(pid)
     record.delete(force=True)
     es.delete(index='records-hep', doc_type='hep', id=pid.object_uuid)
+    es.indices.refresh('records-hep')
```

Comment:
Same with these 2 changes, they are not needed in principle, was trying to debug the problems with `invenio-db[postgresql,versioning]==1.0.0b3`